content/learn/book/getting-started/_index.md

```diff
@@ -38,7 +38,7 @@ Add the bevy crate to your project's Cargo.toml like this:
 
 ```toml
 [dependencies]
-bevy = "0.4" # make sure this is the latest version
+bevy = "0.5" # make sure this is the latest version
```

Comment:
The [setup page](https://bevyengine.org/learn/book/getting-started/setup/) has a reference to bevy 0.4 as well, I think you need to update there as well.